package.json

```diff
@@ -1,22 +1,49 @@
 {
-  "name": "eventhome_api",
-  "version": "1.0.0",
-  "description": "The API for our entire application",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/Event-Booking/EventHome_api.git"
-  },
-  "author": "eventish",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/Event-Booking/EventHome_api/issues"
-  },
-  "homepage": "https://github.com/Event-Booking/EventHome_api#readme",
-  "dependencies": {
-    "dotenv": "^8.1.0"
-  }
+	"name": "eventhome",
```

Comment:
Nice. Wou;ld you please change the `name` back to `eventhome_api` as this is not a package.json for the entire application but for the api.